web/test/pages/onboarding/onboarding-starting.test.tsx

```diff
@@ -274,4 +274,13 @@ describe("onboarding - starting a business", () => {
   describe("validates self-reg step", () => {
     runSelfRegPageTests({ businessPersona: "STARTING", selfRegPage: "4" });
   });
+
+  describe("when onboarding flow is starting", () => {
+    it("routes user to step 2 when query parameter exists and value is starting", () => {
+      useMockRouter({ isReady: true, query: { flow: "starting" } });
+      renderPage({});
+
+      expect(screen.getByTestId("step-2")).toBeInTheDocument();
+    });
+  });
```

Comment:
I think it might be more accurate to put this new test in `onboarding-shared` because as soon as we add more flows, it will no longer apply just to "starting". `onboarding-shared` is where we test all the other query parameters that can apply to the onboarding flow.